# 动作量
SingleAgentControl[] ad_action
int32 action_dim # action_dim=2, 方便解析
